gnucash/report/standard-reports/income-gst-statement.scm

```diff
@@ -33,1792 +26,222 @@
-                             width def:secondary-subtotal-style used-columns)))))
-
-                (if (and secondary-subtotal-pred
-                         (or (not next)
-                             (and next
-                                  (not (secondary-subtotal-pred
-                                        current next)))))
-                    (begin (secondary-subtotal-renderer
-                            table width current
-                            secondary-subtotal-collectors
-                            def:secondary-subtotal-style used-columns export?)
-                           (for-each (lambda (coll) (coll 'reset #f #f))
-                                     secondary-subtotal-collectors)
-                           (if next
-                               (secondary-subheading-renderer
-                                next table width
-                                def:secondary-subtotal-style used-columns)))))
-
-            (do-rows-with-subtotals rest
-                                    table
-                                    used-columns
-                                    width
-                                    multi-rows?
-                                    (not odd-row?)
-                                    export?
-                                    account-types-to-reverse
-                                    primary-subtotal-pred
-                                    secondary-subtotal-pred
-                                    primary-subheading-renderer
-                                    secondary-subheading-renderer
-                                    primary-subtotal-renderer
-                                    secondary-subtotal-renderer
-                                    primary-subtotal-collectors
-                                    secondary-subtotal-collectors
-                                    total-collectors))))
-
-    (let* ((table (gnc:make-html-table))
-           (width (num-columns-required used-columns))
-           (multi-rows? #f) ;disable. (transaction-report-multi-rows-p options))
-           (export?  (transaction-report-export-p options))
-           (account-types-to-reverse '()))
-
-      (gnc:html-table-set-col-headers!
-       table
-       (make-heading-list used-columns calculated-cells options))
-
-      (if (not (null? splits))
-          (begin
-            (if primary-subheading-renderer
-                (primary-subheading-renderer
-                 (car splits) table width def:primary-subtotal-style used-columns))
-            (if secondary-subheading-renderer
-                (secondary-subheading-renderer
-                 (car splits) table width def:secondary-subtotal-style used-columns))
-
-            (do-rows-with-subtotals splits table used-columns width
-                                    multi-rows? #t
-                                    export?
-                                    account-types-to-reverse
-                  
```

Comment:
Am I right that:
- (_ xx) strings are translated in UI and code
- (N_ xx) strings are translated in UI but are unchanged in code, i.e. saved-settings are not translated?
If I'm right then I guess should these option names also be changed to: (N_ "xx")?